File: curso-udemy/curso-algoritmo-completo/Módulo 9 - C++/Exercícios/EXCSOMAIMPAR.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    int a, b, z, somaImpar, i;

    cout << "Digite dois numeros: ";
    cin >> a >> b;
    
    if(a > b){
        z = a;
        a = b;
        b = z;
    }

    somaImpar = 0;
    for(i = a+1; i < b; i++){
        if(i % 2 != 0){
            somaImpar = somaImpar + i;
        }
    }

    cout << "Soma impar = " << somaImpar;

    return 0;
}

File: curso-udemy/curso-algoritmo-completo/Módulo 10 - Java/Exercícios/exercicio_java/src/exercicio_java/EXCSOMAIMPAR.java
package exercicio_java;

import java.util.Locale;
import java.util.Scanner;

public class EXCSOMAIMPAR {
	
public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int a, b, z, i, somaImpar;
		
		System.out.println("Digite dois números: ");
		a = sc.nextInt();
		b = sc.nextInt();
		
		if(a > b) {
			z = a;
			a = b;
			b = z;
		}
		
		somaImpar = 0;
		for(i = a + 1; i < b; i++) {
			if(i % 2 != 0) {
				somaImpar = somaImpar + i;
			}
		}
		System.out.println("Soma dos ímpares = " + somaImpar);
		
		sc.close();
	}

}
